src/backend/core/models.py

```diff
@@ -155,6 +176,195 @@ def get_teams(self):
         return []
 
 
+class BaseAccess(BaseModel):
+    """Base model for accesses to handle resources."""
+
+    resource = None
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+    )
+    team = models.CharField(max_length=100, blank=True)
```

Comment:
What is a team in this context, how do I use this field ?

Reply:
A team is a string e.g. "lasuite".
The user model has a method `get_teams` which returns a list of teams for a user as a list of strings. The idea is that this list of teams be queried for example from a permission application (like Desk) via SCIM or retrieved from an OIDC scope...
If a user has the team `lasuite`, the access given to this team will be available to him/her.
Hope it clarifies!